src/routes/(layout)/user/+page.svelte

```diff
@@ -1,21 +1,39 @@
 <script lang="ts">
 	import UserInfo from '$components/user/UserInfo.svelte';
+	import dayjs from 'dayjs';
 
-	const users: { realname: string; username: string }[] = [];
+	const { data } = $props();
 
-	for (let i = 0; i < 5; i++) {
-		users[i] = {
-			realname: 'John Doe',
-			username: 'Monkey'
-		};
+	function parseDate(createdAt: string): string {
+		const now = dayjs();
+		let diff = now.diff(dayjs(createdAt));
+
+		let days = Math.floor(diff / 86400000);
+		let months = Math.floor(days / 30);
+		let years = Math.floor(months / 12);
+
+		if (years > 0) {
+			return years == 1 ? `${years} year ago` : `${years} years ago`;
+		} else if (months > 0) {
+			return months == 1 ? `${months} month ago` : `${months} moths ago`;
+		}
+
+		return days == 1 ? `${days} day ago` : `${days} days ago`;
```

Comment:
This could be done with dayjs plugins duration + relativeTime

Reply:
For now i would only add relativeTime